cabal.release.project

```diff
@@ -2,4 +2,7 @@ import: project-cabal/pkgs/cabal.config
 import: project-cabal/pkgs/install.config
 import: project-cabal/pkgs/tests.config
 
+package *
+  flags: -arch-native
```

Comment:
Does this make sense? A package might provide a flag called "arch-native" which does something entirely different. I think being more precise `package hashable; flags -arch-native` would be better.

Reply:
Or probably `constraints: hashable -arch-native`

Reply:
Does the `constraints` stanza hold more power than the `flags` one?

Reply:
ok nvm the manual says it's just equivalent syntax.